### DISCUSSION TITLE
Curious, is BLOOM an intended target supported model?

### DISCUSSION THREAD

**Discussion by linuxmagic-mp** [ggml-org/llama.cpp#1567] [Ideas]
Just curious, most of the supported models are variations of Llama, and I was curious if Bloom would be a target supported model.  And if not, curious as to why

**Comment by BetaDoggo** [comment]
llama.cpp will only support llama models, but the [ggml](https://github.com/ggerganov/ggml) library can be used with several different architectures. There's a fork for bloom models called [bloomz.cpp](https://github.com/NouamaneTazi/bloomz.cpp).

**Reply by linuxmagic-mp** [reply]
Thanks for that.. Will look at that one, but of course.. sometimes it is a pet peeve that we can't all work together ;) Be nice if this was unified, and my fault for putting that comment in the llama.cpp discussion :0  But this is one of the more interesting developments I have been researching.. Also really well put together including documentation.
